demos/python_demos/image_inpainting_demo/inpainting.py

```diff
@@ -0,0 +1,102 @@
+"""
+ Copyright (c) 2019 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import os
+import cv2
+import numpy as np
+
+from openvino.inference_engine import IENetwork, IECore
+
+class ImageInpainting(object):
+    def __init__(self, ie, model_path, device='CPU', parts=8, max_brush_width=24, max_length=100, max_vertex=20):
```

Comment:
These defaults aren't really useful, since you expliticly specify all of the parameters anyway.